packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -51,6 +51,7 @@ export interface PanelChromeProps {
    * of showing/interacting with the panel's state
    */
   leftItems?: ReactNode[];
+  displayMode?: 'default' | 'transparent';
```

Comment:
Should we have an `enum` for this?

Reply:
don't think we need one yet, depends on how this evolves in the future